packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -608,7 +608,9 @@ - (NSDictionary *)encodingMap
       NSURLSessionDownloadTask *downloadTask = [downloadTasks firstObject];
       if (downloadTask) {
         [downloadTask cancelByProducingResumeData:^(NSData * _Nullable resumeData) {
-          NSString *data = [[NSString alloc] initWithData:resumeData encoding:NSUTF8StringEncoding];
+
+          NSString *data = [resumeData base64EncodedStringWithOptions:NSDataBase64EncodingEndLineWithLineFeed];
```

Comment:
Is there a reason to use `EndLineWithLineFeed` (as opposed to not adding new lines at all)?